tests/test_parameter_mapping.py

```diff
@@ -209,8 +264,12 @@ def test_parameterized_condition_table_changed_scale(
         - constant override (condition 4)"""
 
         document, model = minimal_sbml_model
-        model.createParameter().setId('overridee')
+        p = model.createParameter()
+        p.setId('overridee')
+        p.setValue(2.0)
         assert petab.get_model_parameters(model) == ['overridee']
+        assert petab.get_model_parameters(model, with_values=True) \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4854269)

Reply:
Why does my local pylint not complain about that when using the same config file?